src/main/java/edu/mbrooks/advancedjava/impl/CollectableImpl.java

```diff
@@ -0,0 +1,72 @@
+/**
+ * Program Name:    Collectable.java
+ * Author:          Michael Brooks
+ * Version:         Version 1.0
+ * Course:          Advanced Java Programming, Summer 2018
+ * Copyright:       (C) 2018 Michael A. Brooks
+ * Description:     Implementation of the collectable interface
+ */
+
+package edu.mbrooks.advancedjava.impl;
+
+import edu.mbrooks.advancedjava.interfaces.Icollectable;
+
+/**
+ *
+ * @author michaelbrooks
+ * This is a class to setup if there is a collection of valuables, or a bunch of trash
+ */
+public class CollectableImpl implements Icollectable {
+
+    String pileOf;
+    boolean getRich;
```

Comment:
this should be private 
also Boolean variables are typically hasXXX or isXXX not sure what getRich (true or false) means.

Reply:
Never saw that before, I like that, makes perfect sense